src/Transports.Includes.props

```diff
@@ -11,15 +9,4 @@
 		<ProjectReference Include="..\ServiceControl.Transports.SqlServer\ServiceControl.Transports.SqlServer.csproj" ReferenceOutputAssembly="false" Private="false" />
 		<ProjectReference Include="..\ServiceControl.Transports.SQS\ServiceControl.Transports.SQS.csproj" ReferenceOutputAssembly="false" Private="false" />
 	</ItemGroup>
-
-	<ItemGroup>
-		<Artifact Include="..\ServiceControl.Transports.ASBS\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\NetStandardAzureServiceBus\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.ASQ\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\AzureStorageQueue\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.Learning\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\LearningTransport\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.Msmq\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\MSMQ\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.RabbitMQ\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\RabbitMQ\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.SqlServer\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\SQLServer\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.SQS\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\AmazonSQS\%(RecursiveDir)" />
-		<Artifact Include="..\ServiceControl.Transports.ASB\bin\$(Configuration)\$(TargetFramework)\**\*.*" DestinationFolder="$(OutputPath)\Transports\AzureServiceBus\%(RecursiveDir)" />
-	</ItemGroup>
 </Project>
```

Comment:
We need these for the 3 instances to have the transport folder so I've created a separate include file for them and added it where necessary.